script.js

```diff
@@ -136,9 +135,10 @@ const generatePlayground = (size) => {
     numbersList[number]++;
```

Comment:
Líbilo by se mi
* buď kód lehce zjednodušit s pomocí `do`/`while` smyčky (aby se neopakoval kód s generováním dvakrát)
* a nebo na to jít opačně, nejprve si připravit "karty" a pak je zamíchat pomocí některého algoritmu na náhodné zamíchání pole, třeba něco jako `let emojiToPutOnCards = shuffle([...emoji.slice(0, size), ...emoji.slice(0, size)]);` (`shuffle` nevymýšlej, ale zkopíruj např. [odtud](https://stackoverflow.com/a/12646864/3603869), část _EDIT: Updating to ES6_)